CHANGELOG.md

```diff
@@ -1,37 +1,52 @@
+## [v0.14.0]
+> June 15th, 2017
+
+- Removed Rollbar in favor of Sentry (Raven is npm module)
+- Removed Hapi in favor of express
+- Removed most transpilation except for transform-flow-strip-types
+- Removed GQL logic
+- Changed ES6 classes to functions that receive options and return a new instance of that entity (workerServer, cronServer, httpServer, cache)
+- Removed CommonJS module loading in favor of require
```

Comment:
require _is_ commonjs (for all intensive porpoises). Did you mean remove es6 module imports?